### PR TITLE
q dns client: new package

### PR DESCRIPTION
**Pull request by brunomiguel**
#### For new package PRs only
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates

### CODE REVIEW

File: q.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    with:
+      repository: https://github.com/natesales/q
+      tag: v${{package.version}}
+      expected-commit: 83c169f40f0895ae4b5b210f20b0a46fda2e0d9d
+
+  - name: Configure and build
+    runs: |
+      mkdir -p "${{targets.destdir}}"/usr/bin
+      # build package as requested by the upstream author
+      go build -ldflags="-s -w -X main.version=$(git describe --long --tags | sed 's/^v//;s/\([^-]*-g\)//;s/-/./g') -X main.commit=$(git rev-parse --verify HEAD) -X main.date=$(date -Idate)"
```
Comment (stormqueen1990) [inline q.yaml:27]: Small suggestion for the version string

```suggestion
      go build -ldflags="-s -w -X main.version=v${{package.version}} -X main.commit=$(git rev-parse --verify HEAD) -X main.date=$(date -Idate)"
```

File: q.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      install -m755 q "${{targets.destdir}}"/usr/bin/
+
+  - uses: strip
+
+update:
+  enabled: true
+  github:
+    identifier: natesales/q
+    strip-prefix: v
```
Comment (stormqueen1990) [inline q.yaml:36]: Another small nit for the update section

```suggestion
    strip-prefix: v
    use-tag: true
```

File: q.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
+package:
+  name: q
+  version: 0.11.4
+  epoch: 0
+  description: "A tiny command line DNS client with support for UDP, TCP, DoT, DoH, DoQ and ODoH."
+  copyright:
+    - license: GPL-3.0
```
Comment (stormqueen1990) [inline q.yaml:7]: It appears the `GPL-3.0` identifier is deprecated, as per https://spdx.org/licenses/

```suggestion
    - license: GPL-3.0-only
```

**Comment by brunomiguel** [comment]
Thank you for the feedback, @stormqueen1990. I just submitted the changes you suggested.

**Review by imjasonh — APPROVED** [review]

**Review by stormqueen1990 — APPROVED** [review]

**Comment by stormqueen1990** [comment]
Thanks for your contribution!

**Pull request merged**